backend/app.hopps.commons/src/main/java/app/hopps/commons/DocumentData.java

```diff
@@ -0,0 +1,6 @@
+package app.hopps.commons;
+
+import java.net.URL;
+
+public record DocumentData(URL internalFinUrl, Long referenceKey, DocumentType type) {
```

Comment:
sollte der record `Data` implementieren?

Reply:
Ist im dem falle nicht wichitg, da er nicht in einem Kogito prozess benutzt wird. Ich brauche das nur bei den Invoice und Receipt data, da sonst der AbstractDataHandler damit nicht klar kommt.